slave.mk

```diff
@@ -1417,13 +1475,16 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 		chmod +x sonic_debian_extension.sh,
 	)
 
+	export RFS_SPLIT_LAST_STAGE="$(ENABLE_RFS_SPLIT_BUILD)"
```

Comment:
Is it required that we have to do `export RFS_SPLIT_FIRST_STAGE=n` here?

Reply:
It's not. I thought it was cleaner than "if condition". If you think that wrapping it into 'if' is better, I can do it this way

Reply:
it's cleaner, but my suggestion is to add 

`export RFS_SPLIT_FIRST_STAGE=n ` here just in case

Reply:
Oh, ok, that makes sense. Added.